third_party/libtock-drivers/src/nfc.rs

```diff
@@ -0,0 +1,144 @@
+use crate::util;
+use core::cell::Cell;
+use libtock_core::{callback, syscalls};
+
+const DRIVER_NUMBER: usize = 0x30003;
+
+mod command_nr {
+    pub const TRANSMIT: usize = 1;
+    pub const RECEIVE: usize = 2;
+    pub const EMULATE: usize = 3;
+    pub const CONFIGURE: usize = 4;
+    pub const FRAMEDELAYMAX: usize = 5;
+}
+
+mod subscribe_nr {
+    pub const TRANSMIT: usize = 1;
+    pub const RECEIVE: usize = 2;
+    pub const SELECT: usize = 3;
+}
+
+mod allow_nr {
+    pub const TRANSMIT: usize = 1;
+    pub const RECEIVE: usize = 2;
+}
+
+pub struct NfcTag {}
+
+impl NfcTag {
+    pub fn enable_emulation() {
+        NfcTag::emulate(true);
+    }
+
+    pub fn disable_emulation() {
+        NfcTag::emulate(false);
+    }
+
+    pub fn emulate(enabled: bool) -> bool {
```

Comment:
I don't understand your public interface. Do you want people to call `emulate` with the correct bool? Or the `enable` and `disable` function? Do you care about the return value, and why is it only returned in `emulate`?